### DISCUSSION TITLE
Espanso Cmdlet not found after installation

### DISCUSSION THREAD

**Discussion by themrhead** [ScoopInstaller/Main#5231] [Q&A]
I recently installed Espanso app using scoop main bucket source. Regardless the success of the installation the Cmdlet is not found when I try to execute it from Windows Powershell, Command Prompt, even after I tried to reload Profile configs. I also noticed the app is running (Process Explorer, Task Manager).
Is there any tip to fix this kind of behavior of app installed?

**Comment by Elkiwa** [comment]
Did you try `espansod` command?
